profile/README.md

```diff
@@ -0,0 +1,15 @@
+# **Welcome to enmeshed**
+
+**Self-determined data exchange for everyone** - We create a secure data exchange and storage opportunity to allow companies and education providers to communicate with individuals (e.g. applicants, employees or students) in an encrypted way 🔐
+
+👉 To get up to speed quickly, start with our [Documentation](https://github.com/nmshd/documentation)
+
+👉 You need more business insights? Visit our [Website](https://enmeshed.de/), [LinkedIn](https://www.linkedin.com/showcase/enmeshed/?viewAsMember=true), [Twitter](https://twitter.com/enmeshed_jssoft) or [Mastodon](https://mastodon.social/@enmeshed)! 
+
+------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Harassment-free experience for everyone - We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community --> 🌈 [Contribution guidelines](https://github.com/nmshd/.github/blob/main/CONTRIBUTING.md) + [Code of conduct](https://github.com/nmshd/.github/blob/main/CODE_OF_CONDUCT.md)
+
+Enmeshed takes the security of our software products and services seriously, including all of the open source code repositories managed through our organization --> 🚓 [Security Policy](https://github.com/nmshd/.github/security/policy)
+
+ ![enmeshed-logo_invertiert klein](https://user-images.githubusercontent.com/128588360/236791065-c04c49ef-a774-4efb-8579-81d54621fd7a.svg)
```

Comment:
2 Sachen:
1.
```suggestion
 ![enmeshed-logo_invertiert small](https://user-images.githubusercontent.com/128588360/236791065-c04c49ef-a774-4efb-8579-81d54621fd7a.svg)
```
2. Ich frage mich gerade, ob es sinnvoll ist, das invertierte Logo zu nutzen. Wenn jemand den light mode hat, sieht er halt die Striche nicht. Andersrum hat man natürlich Probleme, das nicht-invertierte Logo zu sehen, wenn man den Dark Mode hat... Keine Ahnung was hier best practice ist.

Reply:
zu 2. ich glaube man kann logos für dark & light nutzen

Reply:
https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown/

Reply:
@jkoenig134 und @tnotheis Danke für das ganze Feedback! Hab soweit alles eingearbeitet. Nur beim Light Mode / Dark Mode steig ich noch nicht hinter. Hab es im Code eigentlich genauso hinterlegt wie in dem Link von Julian. Allerdings zeigt er immer noch beide Logos gleichzeitig an auch wenn ich zwischen Dark / Light Mode wechsle

@jkoenig134 Hast du eine Idee?

Reply:
ne, du hast nur einen Teil des Tutorials gemacht glaube ich, ich bearbeite es.